code/modules/mob/living/death.dm

```diff
@@ -74,6 +74,16 @@
 
 	if (client)
 		client.move_delay = initial(client.move_delay)
+		if(!suiciding && !SSticker.first_death.len) //suicides don't clog up the list, only BRUTAL deaths here
+			var/list/L = list()
+			L["name"] = name
+			L["role"] = null
+			if(mind.assigned_role)
+				L["role"] = mind.assigned_role
+			L["area"] = "[get_area_name(src, TRUE)] [COORD(src)]"
+			L["damage"] = "<font color='#FF5555'>[bruteloss]</font>/<font color='orange'>[fireloss]</font>/<font color='lightgreen'>[toxloss]</font>/<font color='lightblue'>[oxyloss]</font>/<font color='pink'>[cloneloss]</font>"
+			L["last_words"] = last_words
+			SSticker.first_death = L

```

Comment:
Why does this record in both SSticker and SSblackbox, was this meant to be removed?

Reply:
that's odd. yeah, I guess I forgot to take it out.